cfgov/v1/models/learn_page.py

```diff
@@ -40,7 +40,7 @@ class AbstractFilterPage(CFGOVPage):
     # General content tab panels
     content_panels = CFGOVPage.content_panels + [
         StreamFieldPanel('header'),
-        StreamFieldPanel('content'),
+        # StreamFieldPanel('content'),
```

Comment:
This looks like a copy/paste error that was introduced a long time ago, since the `content` StreamField correctly does not exist here. There must have been an update in their field checking that its finally catching this bug.

e.g. of where content does exist `https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/v1/models/landing_page.py#L18`